src/packages/@ncigdc/modern_components/ClinicalAnalysis/ClinicalVariableCard.js

```diff
@@ -496,55 +503,55 @@ const ClinicalVariableCard: React.ComponentType<IVariableCardProps> = ({
       ),
       ...(variable.active_chart === 'survival'
```

Comment:
maybe this can be fixed along with the other boolean spreads

Reply:
keeping this as a spread operator but modified the syntax - it's adding a key to an object and I want that to be conditional